packages/entities/entities-plugins/src/definitions/schemas/JWT.ts

```diff
@@ -25,7 +25,7 @@ export const jwtSecretSchema: JwtSecretFieldSchema = {
         use to verify the token’s signature.`,
         rows: 10,
         visible: (model) => {
-          return model && model.algorithm && /(ES|RS)[\d]{3}/.test(model.algorithm)
+          return model && model.algorithm && /(RS|PS|Ed)[\dA-Za-z]{3}/.test(model.algorithm)
```

Comment:
Is missing the `ES` expected here?

Reply:
Yes. Please ref here. https://github.com/Kong/kong-ee/blob/master/kong/plugins/jwt/daos.lua#L60-L67

Reply:
@raoxiaoyan I'd suggest to add a comment above this line with a permalink to the code you mentioned to add some context here.

Reply:
@jw-foss Please look at the PR's summary. https://github.com/Kong/public-ui-components/pull/2004#issue-2896491070

Reply:
BTW, it could be more accurate to write regex like this `/^((RS|PS|ES)(256|384|512)|EdDSA)$/`

Reply:
updated